web/ui/react-app/src/pages/Status.tsx

```diff
@@ -55,26 +55,14 @@ export const statusConfig: StatusConfig = {
   droppedAlertmanagers: { skip: true },
 };
 
-const endpointsMemo: { [prefix: string]: string[] } = {};
-
-const Status: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = '' }) => {
-  if (!endpointsMemo[pathPrefix]) {
-    // TODO: Come up with a nicer solution for this?
-    //
-    // The problem is that there's an infinite reload loop if the endpoints array is
-    // reconstructed on every render, as the dependency checking in useFetches()
-    // then thinks that something has changed... the whole useFetches() should
-    // probably removed and solved differently (within the component?) somehow.
-    endpointsMemo[pathPrefix] = ENDPOINTS.map(ep => `${pathPrefix}${ep}`);
-  }
-  const { response: data, error, isLoading } = useFetches<StatusPageState[]>(endpointsMemo[pathPrefix]);
+export const statusRenderer = ({ data, error }: FetchState<StatusPageState>) => {
```

Comment:
I'd call this either `renderStatus`, noun names seem to be more for classes or at least full components?

Reply:
It has a strange status - it is a function (when is used as a children prop in Fetch) but can also act as a Component (see unit tests). Maybe I should unify it and rename it as "Status" as it was before. WDYT?

Reply:
With unify, do you mean merging it together again so it can't be used separately? I do like having it separately, but I think either making it a full separate component or just calling it `status` would be fine? Not sure...

Reply:
Yes, naming it in such a way to become clear that is one thing.